homeassistant/components/zha/sensor.py

```diff
@@ -20,8 +20,10 @@
     DEVICE_CLASS_HUMIDITY,
     DEVICE_CLASS_ILLUMINANCE,
     DEVICE_CLASS_POWER,
+    DEVICE_CLASS_POWER_FACTOR,
     DEVICE_CLASS_PRESSURE,
     DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_FREQUENCY,
```

Comment:
You should try to keep alphabetical order when you see it. `isort` will help with some of it, others need to be fixed manualy.

Also, adding new device classes will need a corresponding PR in developer docs:
https://github.com/home-assistant/developers.home-assistant/blob/master/docs/core/entity/sensor.md